mongonaut/templates/mongonaut/document_detail.html

```diff
@@ -1,41 +1,36 @@
 {% extends "mongonaut/base.html" %}
-{% load url from future %}
 {% load mongonaut_tags %}
 
 {% block breadcrumbs %}
-<li><a href="{% url "index" %}">Mongonaut</a> <span class="divider">/</span></li>
-<li><a href="{% url "document_list" app_label document_name %}">{{ document_name }}s</a> <span class="divider">/</span></li>
-<li><a href="">{{ document_name }} {{ document.id }}</a></li>
+<li class="breadcrumb-item active"><a href="{% url "index" %}">Mongonaut</a></li>
+<li class="breadcrumb-item active"><a href="{% url "document_list" app_label document_name %}">{{ document_name }}s</a></li>
+<li class="breadcrumb-item active"><a href="">{{ document.id }}</a></li>
 {% endblock %}
 
 {% block content %}
 
-<h1>{{ document_name }} Detail</h1>
 
+    <div class="col-sm-12" id="document-actions">
+        <h1>{{ document_name }} Detail</h1>
 
-<div class="row" >
-    <div class="span12" id="document-actions">
         <p>
-            <div class="btn-group">
                 {% if has_edit_permission %}            
-                    <a class="btn btn-primary" href="{% url "document_detail_edit_form" app_label document_name document.id %}">
-                        <i class="icon-pencil icon-white"></i> Edit
+                    <a class="btn btn-primary" style="margin-right:5px"  href="{% url "document_detail_edit_form" app_label document_name document.id %}">
```

Comment:
I'd prefer to avoid the inline style. Makes it harder for customization by consumers.